app/containers/Applications/ApplicationWizardConfirmation.tsx

```diff
@@ -56,13 +58,17 @@ export const ApplicationWizardConfirmationComponent: React.FunctionComponent<Pro
           pathname: '/complete-submit'
         }}
       >
-        <Button
-          className="float-right"
-          style={{marginTop: '10px'}}
-          onClick={submitApplication}
-        >
-          Submit Application
-        </Button>
+        {props.is_analyst ? (
+          ''
+        ) : (
+          <Button
+            className="float-right"
+            style={{marginTop: '10px'}}
+            onClick={submitApplication}
+          >
+            Submit Application
+          </Button>
+        )}
```

Comment:
If we're reusing this component in places where we view the application, I'd rather move the submit button out of it than have it render based on a prop